example/issue_comment.go

```diff
@@ -65,16 +65,21 @@ func (h *PRCommentHandler) Handle(ctx context.Context, eventType, deliveryID str
 
 	repoOwner := repo.GetOwner().GetLogin()
 	repoName := repo.GetName()
-	author := event.GetComment().GetUser()
+	author := event.GetComment().GetUser().GetLogin()
 	body := event.GetComment().GetBody()
 
+	if author[len(author)-5:] == "[bot]" {
```

Comment:
This avoids the infinite-loop, where the bot sees a comment, posts a reply comment, and then replies to that comment, on and on forever